### PR TITLE
msamprz/fix-candle-chart-legend

### PR DESCRIPTION
**Pull request by msamprz**

### CODE REVIEW

File: src/javascript/app/common/chart_settings.js [hunk 1]
```diff
@@ -41,9 +41,7 @@ const ChartSettings = (() => {
         txt_subtitle = (params.is_chart_delayed ? labels.delay : '') +
             (params.is_forward_starting ? labels.purchase_time : '') +
             (params.is_sold_before_start ? '' : start_time) +
-            (history ? ((params.is_sold_before_start || params.is_tick_trade) ? '' : labels.entry_spot) : '') +
```
Comment (negarn) [inline src/javascript/app/common/chart_settings.js]: this also removes it from tick trades, we only want to know why it's showing for candles and remove it from there

File: src/javascript/app/pages/trade/charts/highchart.js [hunk 2]
```diff
@@ -261,7 +262,7 @@ const Highchart = (() => {
 
                         // don't draw start time for contracts that are sold before contract starts
                         if (sell_time < start_time) {
-                            HighchartUI.updateLabels(chart, getHighchartLabelParams());
+                            HighchartUI.updateLabels(chart, getHighchartLabelParams(null, !!history));
```
[the remark this answers is not in the document]
Comment (msamprz) [reply src/javascript/app/pages/trade/charts/highchart.js]: right, of course. please check [#6a6fca5](https://github.com/binary-com/binary-static/commit/6a6fca5e9f8153c85ed7958e329c91f35cf00d9b)

File: src/javascript/app/pages/trade/charts/highchart.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ const addComma         = require('../../../common/currency').addComma;
 const localize         = require('../../../../_common/localize').localize;
 const State            = require('../../../../_common/storage').State;
 const getPropertyValue = require('../../../../_common/utility').getPropertyValue;
+const isEmptyObject = require('../../../../_common/utility').isEmptyObject;
```
Comment (negarn) [inline src/javascript/app/pages/trade/charts/highchart.js]: spacing

Comment (msamprz) [reply src/javascript/app/pages/trade/charts/highchart.js]: noted and addressed, please check [#796ab91](https://github.com/binary-com/binary-static/commit/796ab91735b5c749bd80fda5fda969a80bd8db04)

**Pull request merged**